### ISSUE TITLE
Fields Needed in Dudupe?

### ISSUE THREAD

**Issue by guyiac** [progressivetech/net.ourpowerbase.sumfields#68]
Since Summary Fields are calculated fields, it doesn't seem logical to me that they should show up in dedupe screens to merge. Is there a way to prevent them from appearing there? There is no real harm having them, but if a number of them are activated then it clutters the screen, and just doesn't make sense logically.

**Comment by jmcclelland** [comment]
Agreed! I'm not sure there are any hooks to hide fields on the screen though. Do you know of any way to do that?

**Comment by guyiac** [comment]
LOL...unfortunately no. I'm pretty good to really good on the workflow side, and am always up to do testing and give feedback in the UI, but the programming side is not my area of expertise. Sorry! I'll see if I can get someone to take a look at some point though, now that I know you are in agreement conceptually. Thanks Jamie!
